flink-runtime/src/main/java/org/apache/flink/runtime/shuffle/ShuffleDescriptor.java

```diff
@@ -41,14 +41,14 @@
 	 * that the producer of the partition (consumer input channel) has not been scheduled
 	 * and its location and other relevant data is yet to be defined.
 	 * To proceed with the consumer deployment, currently unknown input channels have to be
-	 * marked with placeholders which are special implementation of {@link ShuffleDescriptor}:
+	 * marked with placeholders which are special implementation of shuffle descriptor:
 	 * {@link UnknownShuffleDescriptor}.
 	 *
 	 * <p>Note: this method is not supposed to be overridden in concrete shuffle implementation.
 	 * The only class where it returns {@code true} is {@link UnknownShuffleDescriptor}.
 	 *
 	 * @return whether the partition producer has been ever deployed and
-	 * the corresponding {@link ShuffleDescriptor} is obtained from the {@link ShuffleMaster} implementation.
+	 * the corresponding shuffle descriptor is obtained from the {@link ShuffleMaster} implementation.
```

Comment:
Are there any concerns for this change?

Reply:
there are some check style warnings in Idea IDE to not use self links